proposals/EFIP-6.md

```diff
@@ -0,0 +1,33 @@
+# [EFIP-6] Fine-Grained Timelock
```

Comment:
Since the implementation adds a new `pauser` contract as a single entry point for the protocol pause/unpause, you can update the EFIP slightly :)